code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -38,6 +38,7 @@
 /mob/living/simple_animal/hostile/megafauna/Initialize(mapload)
 	. = ..()
 	apply_status_effect(STATUS_EFFECT_CRUSHERDAMAGETRACKING)
+	add_trait(TRAIT_NO_TELEPORT, MEGAFAUNA_TRAIT)
```

Comment:
Dose this affect BDM or Bubblegum?

Reply:
Should affect all of them by inheritance, admittedly only tested with bubblegum and drake

Reply:
Ok, so dose it mess with there teleports? Or will they be fine, IIRC blood drunk miner dose a new/loc call and I think Bubble gum is able do a teleport move at you when crushing, unless im wrong*

Reply:
Those special attacks don't use teleport on bubblegum, I'll just check on miner.

Reply:
Yeah, neither should be affected.

Reply:
Noted.

Reply:
Yeah this is checking if the portal will teleport after something has passed onto it. It shouldn’t affect forcemove in every way (I should know, my new pr does a ton of new teleporter shit)